pkg/libovsdb/schema.go

```diff
@@ -736,6 +738,7 @@ func (columnSchema *ColumnSchema) ValidateMap(value interface{}) bool {
 }
 
 func (columnSchema *ColumnSchema) Validate(value interface{}) bool {
+	klog.V(5).Infof("validate columnSchema : %v value: %v", columnSchema, value)
```

Comment:
remove: this is library code, it should not log message, this is for the caller program to decide